### PR TITLE
Fix/migration

### PR DESCRIPTION
**Pull request by mfshao**
### Bug Fixes
- fixed a bug causing DB migration to crash for special string combinations

### CODE REVIEW

File: tests/test_migrations.py [hunk 1]
```diff
@@ -253,12 +253,13 @@ async def test_6819874e85b9_upgrade():
     old_metadata = {
         "foo": "bar",
         "bizz": "buzz",
+        "special": "50% for",
```
Comment (mfshao) [inline tests/test_migrations.py:256]: This will fail before the fix, with error from sqlalchemy `typeerror dict is not a sequence`

Comment (mfshao) [reply tests/test_migrations.py:256]: and strangely I don't need to escape `%` here https://github.com/uc-cdis/metadata-service/blob/fix/migration/tests/test_migrations.py#L12

is it because sqlalchemy is handling `INSERT` and `UPDATE` differently? 🤔 @paulineribeyre @george42-ctds

Comment (paulineribeyre) [reply tests/test_migrations.py:256]: i think it works in the tests because we are inserting with `db.scalar(db.text(...))`.

In the migration we are using `connection.execute(...)` but i think it would work if we used something like `connection.execute(sa.text(...))`.

From [this post](https://www.codesprogram.com/questions/58644/TypeError-dict-is-not-a-sequence?s=answer&aid=4103) - i haven't tested it

Comment (mfshao) [reply tests/test_migrations.py:256]: I actually have tried the `sa.text()` solution, the problem with that is it will translate a string like this `[Pain Med. 2011; 12(4):657-67. 2013; 14(10):1534-47]` into using bindparams like `[Pain Med. 2011; 12(4)%(657-67.)s 2013; 14(10)%(1534-47)s]` by default, and then the operation will fail. So instead of escaping `%`, I now have to escape `:` in `escape()`

But if you think using `sa.text()` is a more generic solution I can also modify my PR with it

Comment (paulineribeyre) [reply tests/test_migrations.py:256]: 🤔 oh ok, then maybe we can keep your fix but add the edge case with `:` in the test too?

Comment (mfshao) [reply tests/test_migrations.py:256]: actually given another thought, I think using `sa.text()` is more generic, I'll update my the migration script and tests

**Review by paulineribeyre — APPROVED** [review]

**Review by george42-ctds — APPROVED** [review]
Looks great!
